core/controllers/editor_test.py

```diff
@@ -1397,8 +1397,12 @@ def test_for_assign_role_for_exploration(self):
         self.login(self.COLLABORATOR4_EMAIL)
         csrf_token = self.get_new_csrf_token()
 
-        # Since check edit_activity_always return False, therefore
-        # user will behave as voiceover artist.
+        reader_dict = self.get_json(
+            '%s/%s' % (feconf.EXPLORATION_DATA_PREFIX, exp_id))
+        self.assertNotIn('State 4', reader_dict['states'])
+
+        # Since check_can_edit_activity always return False, therefore
```

Comment:
therefore --> the

("Since" already conveys "therefore".)

Reply:
edited. Thanks for pointing it out.